workers/data_refinery_workers/processors/create_compendia.py

```diff
@@ -189,11 +198,13 @@ def _perform_imputation(job_context: Dict) -> Dict:
     combined_matrix_zero = row_col_filtered_combined_matrix_samples
```

Comment:
`microarray_expression_matrix`
`filtered_rnaseq_matrix_plus_one`
`combined_matrix`
`row_filtered_combined_matrix`
Do these get used beyond this point? If not, those should also be pretty large.


It looks like `row_col_filtered_combined_matrix_samples.index` and `row_col_filtered_combined_matrix_samples.columns` get used later. I think you could probably move the lines where you snag that info here and then delete the `row_col_filtered_combined_matrix_samples` object. The axis labels should be a lot smaller than the matrix itself, if I'm understanding that correctly.